### PR TITLE
Prevent chronoshifting an empty selection

### LINKED ISSUE #16425: Chronoshift has bogus terrain checks
One can use an empty Chronoshift only if the terrain matches at the starting location and exit one. This means that if there's one road / beach / rubble / ore / gems tile in the mix at the starting location it also has to be in the exact same place on the exit location otherwise Chronosphere refuses to work. IMO Chronospere shouldn't fire if it's empty

### PR DESCRIPTION
**Pull request by abcdefg30**
Closes #16425.

### CODE REVIEW

File: OpenRA.Mods.Cnc/Traits/SupportPowers/ChronoshiftPower.cs [hunk 1]
```diff
@@ -266,15 +266,13 @@ protected override IEnumerable<IRenderable> RenderAboveShroud(WorldRenderer wr,
 				var xy = wr.Viewport.ViewToWorld(Viewport.LastMousePos);
 				var palette = wr.Palette(power.Info.IconPalette);
 
-				// Destination tiles
```
Comment (matjaeck) [inline OpenRA.Mods.Cnc/Traits/SupportPowers/ChronoshiftPower.cs]: I find this comment and the one below useful.

**Review by ghost — COMMENTED** [review]
LGTM.

**Comment by teinarss** [comment]
Needs rebase!

**Comment by abcdefg30** [comment]
Rebased.

**Review by teinarss — APPROVED** [review]

**Comment by teinarss** [comment]
https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/64bc9cfd322dc4e51bf7f59acda15848d98d7b3d...fb8357e9a493c3cb8fc0dd4802e087360183d9f3

**Comment by pchote** [comment]
This was picked to prep as 3ec995258d9bfe5cdc30943edeb3e328d505c084, changelog https://github.com/OpenRA/OpenRA/wiki/Changelog/_compare/5defc2cbf6a13d0a6d9d28af8ae58db2543ddf71

**Pull request merged**